test/spec/MessagesSpec.js

```
@@ -88,6 +98,7 @@ describe('Messages should work', function () {
     const selector = findSelect(entry);
     expect(selector).to.exist;
     expect(selector.length).to.equal(2);
+    print(selector)
```

ESLint

[error] Unexpected use of 'print'.

(no-restricted-globals)